test/behavior/var_args.zig

```diff
@@ -105,6 +105,7 @@ test "simple variadic function" {
         return error.SkipZigTest;
     }
     if (builtin.cpu.arch == .x86_64 and builtin.os.tag == .windows) return error.SkipZigTest; // TODO
+    if (builtin.cpu.arch == .s390x and builtin.zig_backend == .stage2_llvm) return error.SkipZigTest; // TODO
```

Comment:
Would be good to file an issue tracking the failing C ABI tests you find for s390x.

Reply:
will make sure to do that as followup